src/jltypes.c

```diff
@@ -2044,8 +2044,9 @@ void jl_init_types(void)
     jl_code_info_type =
         jl_new_datatype(jl_symbol("CodeInfo"), core,
                         jl_any_type, jl_emptysvec,
-                        jl_perm_symsvec(9,
+                        jl_perm_symsvec(10,
                             "code",
+                            "signature_for_inference_heuristics"
```

Comment:
shout out to @maleadt for pointing out the missing comma here that was causing my bootstrap problems



![](http://i0.kym-cdn.com/photos/images/newsfeed/000/698/489/2f7.png)

Reply:
been there, buddy